submission/persistence/src/main/kotlin/ac/uk/ebi/biostd/persistence/integration/SubmissionSqlQueryService.kt

```diff
@@ -35,6 +35,8 @@ class SubmissionSqlQueryService(
     override fun getExistingFolder(accNo: String): File? =
         findSubmission(accNo)?.let { folderResolver.getSubmissionFolder(it.relPath).toFile() }
 
+    override fun getOwner(accNo: String): String? = findSubmission(accNo)?.owner?.email
```

Comment:
Why is the owner nullable? I don't think we can have a submission without owner

Reply:
`in case submission does not exist `